config/quintype-build.js

```diff
@@ -1,11 +1,22 @@
 const { produce } = require("immer");
 const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
+const fs = require("fs");
 
 module.exports = {
   modifyWebpackConfig: function({ defaultConfig }) {
     const config = produce(defaultConfig, function(draft) {
       draft.node = { Buffer: false };
       draft.entry["font"] = "./app/client/font.js";
+      const dir = "./app/assets/fonts";
+      try {
+        fs.readdirSync(dir).forEach(file => {
+          const fullPath = `${dir}/${file}`;
+          draft.entry[file] = fullPath;
+        });
+      } catch (error) {
+        console.log("Error !!!", error);
```

Comment:
Why is try/catch needed here?

Reply:
No need here, but added for sefty, by any chance if some error occur while reading the files , it should able to handle it.